FluentTerminal.App.ViewModels/MainViewModel.cs

```diff
@@ -214,6 +218,34 @@ public Task AddConfigurableTerminal()
             });
         }
 
+        public Task AddRemoteTerminal()
+        {
+            return ApplicationView.RunOnDispatcherThread(async () =>
+            {
+                var connectionInfo = await _dialogService.ShowSshConnectionInfoDialogAsync();
+
+                if (connectionInfo == null)
+                {
+                    if (Terminals.Count == 0)
+                    {
+                        await ApplicationView.TryClose();
+                    }
+
+                    return;
+                }
+
+                var profile = new ShellProfile
+                {
+                    Arguments = $"-p {connectionInfo.Port:#####} {connectionInfo.Username}@{connectionInfo.Host}",
+                    Location = @"C:\Windows\System32\OpenSSH\ssh.exe",
```

Comment:
Do all versions of Windows which FluentTerminal supports include OpenSSH? If not, there should probably be some graceful handling of this (i.e. explain to the user why it doesn't work).

Reply:
Seems like openssh.exe was added in 1709, which is also our current minimum version

Reply:
OpenSSH is a "Windows Feature", and can be installed if already not. (The same as with WSL.) But yes, I agree that we should handle this! Will create an issue for this one.